src/editor/editor.tsx

```diff
@@ -135,13 +135,14 @@ function Editor(props: any) {
     const urlParams = qs.parse(props.location.search, { ignoreQueryPrefix: true });
     const urlSpec = urlParams?.spec ? JSONUncrush(urlParams.spec as string) : null;
     const urlGist = urlParams?.gist ?? null;
+    const urlExampleIndex = urlParams?.example ? examples.map(d => d.id).indexOf(urlParams.example as string) : null;
```

Comment:
Currently, when a wrong example id is used, the site becomes entirely empty. How about changing this part to show a default (e.g., first) example if `example` does not exist?